### ISSUE TITLE
Get connection host/port

### ISSUE THREAD

**Issue by tresf** [qzind/tray#181]
We can specify a host using `qz.websocket.connect( { host: '192.168.1.71' })` but we offer no way to retrieve the connection values back such as `host` and `port`.

**Comment by bberenz** [comment]
Implemented under c1a37931340caa409c016dcb53dddb50bfdde577

**Comment by tresf** [comment]
```js
var conn = qz.websocket.getConnectionInfo();
console.log(conn.host + ":" + conn.port);
```

Returns:
```js
{ host: "localhost",  port: 8283, socket, "ws" }
```

Closed via consolidated commit cb3b494.
